src/pages/home/ThankYouAlert.tsx

```
@@ -0,0 +1,31 @@
+    >
+      <div className="flex flex-col gap-2 !pl-1">
+        <AlertTitle>{t("thankYou.title")}</AlertTitle>
+        <AlertDescription>
+          <Trans
+            i18nKey="thankYou.description"
+            components={{
+              a: (
+                <a
+                  href="https://github.com/sumi-0011/Thumbnail-Maker/issues/new"
+                  className="underline"
+                />
```

Biome (2.3.13)

[error] Provide screen reader accessible content when using `a` elements.

All links on a page should have content that is accessible to screen readers.
Accessible content refers to digital content that is designed and structured in a way that makes it easy for people with disabilities to access, understand, and interact with using assistive technologies.
Follow these links for more information,
 WCAG 2.4.4
 WCAG 4.1.2

(lint/a11y/useAnchorContent)